tutorials/evoked/plot_20_visualize_evoked.py

```diff
@@ -0,0 +1,249 @@
+"""
+.. _tut-visualize-evoked:
+
+Visualizing Evoked data
+=======================
+
+This tutorial shows the different visualization methods for
+:class:`~mne.Evoked` objects.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need:
+"""
+
+import os
+import numpy as np
+import mne
+
+###############################################################################
+# Instead of creating the :class:`~mne.Evoked` object from an
+# :class:`~mne.Epochs` object, we'll load an existing :class:`~mne.Evoked`
+# object from disk. Remember, the :file:`.fif` format can store multiple
+# :class:`~mne.Evoked` objects, so we'll end up with a :class:`list` of
+# :class:`~mne.Evoked` objects after loading. Recall also from the
+# :ref:`tut-section-load-evk` section of :ref:`the introductory Evoked tutorial
+# <tut-evoked-class>` that the sample :class:`~mne.Evoked` objects have not
+# been baseline-corrected and have unapplied projectors, so we'll take care of
+# that when loading:
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_evk_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis-ave.fif')
+evokeds_list = mne.read_evokeds(sample_data_evk_file, baseline=(None, 0),
+                                proj=True, verbose=False)
+# show the condition names
+for e in evokeds_list:
+    print(e.comment)
+
+###############################################################################
+# To make our life easier, let's convert that list of :class:`~mne.Evoked`
+# objects into a :class:`dictionary <dict>` (make sure the order is correct!).
+# We'll use ``/``-separated dictionary keys to encode the conditions, like is
+# often done when epoching, because some of the plotting methods can take
+# advantage of that style of coding.
+
+conds = ('aud/left', 'aud/right', 'vis/left', 'vis/right')
+evks = dict(zip(conds, evokeds_list))
+
+###############################################################################
+# Plotting signal traces
+# ^^^^^^^^^^^^^^^^^^^^^^
+#
+# The most basic plot of :class:`~mne.Evoked` objects is a butterfly plot of
```

Comment:
maybe you can add a note why it's called a butterfly plot.

```
.. note:: Such plot is called a butterfly plot because a source in the brain generates
simultaneously positive and negative values on sensors so the lines deviate positively
and negatively producing what can appear as wings of a butterfly.
```